packages/core/src/components/axes/axis.ts

```diff
@@ -186,20 +193,24 @@ export class Axis extends Component {
 				if (!scale.ticks(numberOfTicks).length) {
 					axis.tickValues([]);
 				} else {
-					const tickValues = scale
-						.nice(numberOfTicks)
-						.ticks(numberOfTicks);
+					const addSpaceOnEdges = Tools.getProperty(
+						options,
+						"timeScale",
+						"addSpaceOnEdges"
+					);
+
+					let tickValues;
+					if (addSpaceOnEdges) {
+						tickValues = scale
+							.nice(numberOfTicks)
+							.ticks(numberOfTicks);
+					} else {
+						tickValues = scale.ticks(numberOfTicks);
+					}
 
 					// Remove labels on the edges
 					// If there are more than 2 labels to show
-					if (
-						Tools.getProperty(
-							options,
-							"timeScale",
-							"addSpaceOnEdges"
-						) &&
-						tickValues.length > 2
-					) {
+					if (addSpaceOnEdges && tickValues.length > 2) {
```

Comment:
prettier seems to have added a lot of changes, this is what I've changed logic wise in the code

Reply:
@natashadecoste @Donisius @cal-smith